app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,8 @@
+class TagsController < ApplicationController
+
+  def show
+    @tag = Tag.find_by(name: params[:name])
+    render json: @tag, params[:name]
```

Comment:
Forgot to mention this on the last pass ... rather than `params[:name]`, you'd need to include 'notes.tags' to force your serializer to go one layer deeper.